lib/smart_answer_flows/maternity-paternity-calculator/adoption_calculator_flow.rb

```diff
@@ -254,12 +200,8 @@ def define
           option :monthly
 
           on_response do |response|
-            calculator.pay_pattern = response
-          end
-
-          calculate :calculator do |response|
-            calculator.pay_method = response
-            calculator
```

Comment:
Woah this is odd.

Reply:
Absolutely. The relationship between `pay_method` and `pay_pattern` confused the hell out of me. Here they appear to be the same, but in other parts of the flow/logic they are most definitely not the same.